geth/testing/testing.go

```diff
@@ -110,15 +113,23 @@ func FirstBlockHash(require *assertions.Assertions, nodeManager common.NodeManag
 	require.Equal(expectedHash, firstBlock.Hash.Hex())
 }
 
+// MakeTestNodeConfig returns the default params.NodeConfig for testing.
 func MakeTestNodeConfig(networkID int) (*params.NodeConfig, error) {
+	testDir := filepath.Join(TestDataDir, TestNetworkNames[networkID])
+
+	if runtime.GOOS == "windows" {
+		testDir = filepath.ToSlash(testDir)
+	}
```

Comment:
On windows, the path produced is in the form 'C:/User/...', but windows uses backslash and therefore this will cause an error when attempting to run tests. This resolves that and allows the retrieval of the directory appropriately on windows as well.